com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/bundle/test/BundlerTest.java

```diff
@@ -532,15 +545,16 @@ public void testAbsoluteCustomResourcePath() throws IOException, ConfigurationEx
                 expectedFiles.add("assets/game.dmanifest");
                 expectedFiles.add("assets/game.projectc");
                 expectedFiles.add("assets/game.public.der");
-                expectedFiles.add("META-INF/CERT.RSA");
-                expectedFiles.add("META-INF/CERT.SF");
+                expectedFiles.add("META-INF/BNDLTOOL.SF");
+                expectedFiles.add("META-INF/BNDLTOOL.RSA");
                 expectedFiles.add("META-INF/MANIFEST.MF");
                 expectedFiles.add("res/drawable-hdpi-v4/icon.png");
                 expectedFiles.add("res/drawable-ldpi-v4/icon.png");
                 expectedFiles.add("res/drawable-mdpi-v4/icon.png");
                 expectedFiles.add("res/drawable-xhdpi-v4/icon.png");
                 expectedFiles.add("res/drawable-xxhdpi-v4/icon.png");
                 expectedFiles.add("res/drawable-xxxhdpi-v4/icon.png");
+                expectedFiles.add("res/xml/splits0.xml");
```

Comment:
Added by bundletool when creating a universal.apk from .aab file. Not exactly sure what it is, but the file is small.